client/pages/form-pages/application-table.tsx

```diff
@@ -0,0 +1,151 @@
+import { Button, SearchBar } from '@components';
+import { Pagination } from '../../components/form/Pagination';
+import { useState } from 'react';
+import initialValues from './example_submission_data.json';
+
+export interface ActivitiesGapProps {
```

Comment:
Can be removed along with the commented-out things!

Reply:
Removed commented out code.